### ISSUE TITLE
How to get list of signed in SIP account and delete/sign out completely an specified account

### ISSUE THREAD

**Issue by nhoxbypass** [BelledonneCommunications/linphone-android#85]
Currently, when i use your code to delete account ``prefs.deleteAccount()`` but not kill the org.linphone process, it did not work for me. The auth is still remain in system, and others can also call to this account.

Another problem is when I log in 2 account, then i delete the second account, then I try to get auth list by calling ``LinphoneManager.getLc().getAuthInfosList()`` or ``getProxyConfigList()`` to check number of account still exist in system, it only show the SECOND account, which should be deleted, but not the first one. But anyway all of them still can receive incoming call for it not delete completely from the system as I mentioned above.

In conclusion I want to know How to get list of signed in SIP account and delete/sign out completely an specified account? Thank you!

**Comment by Thucnv** [comment]
Worked for me:
 - unRegister Server with setExpire = 0
       if(prxCfg != null) {
		prxCfg.edit();
		prxCfg.setExpires(0);
		prxCfg.done();
		getLc().refreshRegisters();
	}
 - RemoveProxyconfig from ProxyConfigList
        getLc().removeProxyConfig(proxyCfg)

**Comment by nhoxbypass** [comment]
@Thucnv  how about AuthInfo list? Sometimes i remove proxyConfig base on index, but the AuthInfo of this account still there

**Comment by Thucnv** [comment]
LinphoneProxyConfig prxCfg = getProxyConfig(index);
		LinphoneAddress addr = LinphoneCoreFactory.instance().createLinphoneAddress(prxCfg.getIdentity());
		LinphoneAuthInfo authInfo = getLc().findAuthInfo(addr.getUserName(), null, addr.getDomain());
getLc().removeAuthInfo(authInfo);

**Comment by nhoxbypass** [comment]
Should I manually remove AuthInfo? I see when Linphone delete an account, it does not remove AuthInfo directly.

**Comment by Thucnv** [comment]
I think unnecessary. In my case, i don't remove authinfo, i see its auto remove after some minute.

**Comment by nhoxbypass** [comment]
Ok I will try. Thank you.
